NotebookTestScript.dib

```diff
@@ -0,0 +1,143 @@
+#!markdown
+
+# Execute the following cell.
+
+You'll notice a counter that appears and starts incrementing.  After a second or two, stop the cell.  Your output should look similar to:
+
+```
+5
+Error: Comamnd cancelled
+```
+
+#!csharp
+
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive;
+
+var output = string.Empty.Display("text/plain");
+var counter = 1;
+while (!KernelInvocationContext.Current.CancellationToken.IsCancellationRequested)
+{
+    await Task.Delay(500, KernelInvocationContext.Current.CancellationToken);
+    output.Update($"{counter++}");
+}
+
+#!markdown
+
+# Execute the next two cells.
+
+#!csharp
+
+var x = 123;
+
+#!html
+
+<div id="output"></div>
+
+#!markdown
+
+# Execute the next cell.  After execution, the output immediately above this should read:
+
+```
+The value is 123.
+```
+
+#!javascript
+
+const x = await interactive.csharp.getVariable("x");
+document.getElementById("output").innerText = `The value is ${x}.`;
+
+#!markdown
+
+# Execute the next cell and verify the following error appears:
+
+```
+Error: input.fsx (1,11)-(1,12) parse error Unexpected token '+' or incomplete expression
+```
+
+#!fsharp
+
+let x = 1 +
+
+#!markdown
+
+# Check this manual scenario:
+
+1. `Ctrl+Shift+P` => ".NET Interactive: Create new blank notebook"
+2. Select "Create as .dib"
+3. Select "F#"
+4. You should have a new empty notebook named something like "Untitled-1.dib" with a single cell of type "F# (Interactive)".
+5. Set the cell contents to `1+1` and execute.
+6. Verify output of `2`.
+7. Save to disk.
+8. Open that file in notepad.
+9. The contents should look like this:
+
+   ```
+   #!fsharp
```

Comment:
Note that this is indented several spaces.  This is required so our parser doesn't think this is a new cell.